components/Header/index.js

```diff
@@ -15,6 +15,8 @@ function Header() {
         <h1>Artikel editieren:</h1>
```

Comment:
I think it would be a little prettier if you remove the : from your headline, so its just Artikel editieren.